pkg/lib/math/math.go

```diff
@@ -0,0 +1,16 @@
+package math
```

Comment:
yeah the plan is to break things out of `system` package slowly to reduce falling into circular dependencies. I can replace min/max in a separate PR to avoid adding more to this one